### PR TITLE
get rid of separated add_label calls

### PR DESCRIPTION
**Pull request by ataymano**

### CODE REVIEW

**Comment by jackgerrits** [comment]
Since you are getting rid of these calls can you provide guidance on what the correct way to add a label is?

**Comment by ataymano** [comment]
> Since you are getting rid of these calls can you provide guidance on what the correct way to add a label is?

Hm, it looks like having separate add_label requires additional documentation, while without it interface is more consistent with any other entry points and there is no specific instructions except generic input format guide: https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Input-format

Are you suggesting to leave this link in the header file?

**Comment by jackgerrits** [comment]
Oh no that's fine if it just has to be part of the text input. Wasn't sure if there was a function to replace that.

File: vowpalwabbit/vwdll.cpp [hunk 1]
```diff
@@ -219,17 +219,6 @@ VW_DLL_MEMBER size_t VW_CALLING_CONV VW_HashFeatureStaticA(const char * s, unsig
   return VW::hash_feature_static(str, u, hash, num_bits);
 }
 
-VW_DLL_MEMBER void  VW_CALLING_CONV VW_AddLabel(VW_EXAMPLE e, float label, float weight, float base)
```
Comment (lokitoth) [inline vowpalwabbit/vwdll.cpp]: This may be breaking API change. We should probably go through a deprecation cycle for it.

**Comment by JohnLangford** [comment]
It seems like this patch is just removing add_label, but I'm unclear on what's motivating that.  Why do that?

**Pull request closed without merging**